### PR TITLE
Blank mission support for dependency editor

### PR DESCRIPTION
**Pull request by invincibleJai**
Tracks : 

- https://github.com/openshiftio/openshift.io/issues/3244

Depends on : 

- https://github.com/fabric8-analytics/fabric8-analytics-dependency-editor/pull/105

### CODE REVIEW

File: src/app/launcher/service/dependency-editor.service.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { Observable } from 'rxjs';
+
+import { DependencyCheck } from '../model/dependency-check.model';
+
+/**
+ * Abstract dependency service provided to ensure consumer implements this pattern
+ */
+export abstract class DependencyEditorService {
+    /**
+     * Abstract service to get information about the boosters selected by the user.
```
Comment (jyasveer) [inline src/app/launcher/service/dependency-editor.service.ts]: comments should be updated

Comment (invincibleJai) [reply src/app/launcher/service/dependency-editor.service.ts]: done.

**Comment by invincibleJai** [comment]
@edewit pls review .

**Pull request merged**